setup.py

```diff
@@ -313,7 +313,7 @@ def geometry_extensions():
     packages=find_packages(),
     cmdclass={**versioneer.get_cmdclass(), "build_ext": build_ext},
     install_requires=[
-        "numpy>=1.25.0,<3",
+        "numpy>=2.0.0,<3",
```

Comment:
```suggestion
        "numpy=2",
```
A little simpler, am I forgetting a reason this wouldn't work? We ought to be well past pre-release versions being pulled down

Reply:
Would have to use `~=2.0`. Single equal sign is conda syntax, iirc. Also can't use a single segment version number for that.

https://packaging.python.org/en/latest/specifications/version-specifiers/#examples
https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release

Reply:
`numpy~=2.0` gives us `numpy>=2.0.0,<3` But i suppose the extra `.0` is giving you grief, haha.

Reply:
I can live with that!